### ISSUE TITLE
Planning: converting all the simple site editor tests to use Gutenberg Blocks

### ISSUE THREAD

**Issue by alisterscott** [Automattic/wp-e2e-tests#1620]
Customers with simple sites can currently opt-in their site to use Gutenberg which is a re-skinned wp-admin editor directly linked from Calypso

This is an issue to plan and track progress of this work as it's quite large to ensure all our editor tests work on Gutenberg for simple sites.

I will break this down into steps which we can split across @Automattic/flowpatrol 

- [ ] Create a new simple test site that we can use for testing gutenberg editor separately to the existing test site which will continue to use Calypso
- [ ] Create a copy of the post editor spec that only runs on simple sites and uses new site specified above
- [ ] Create a copy of the page editor spec that only runs on simple sites and uses new site specified above
- [ ] Update "Public Posts: Preview and Publish a Public Post"
- [ ] Update "Basic Public Post"
- [ ] Update "Check Activity Log for Public Post"
- [ ] Update "Schedule Basic Public Post"
- [ ] Update "Private Posts"
- [ ] Update "Password Protected Posts"
- [ ] Update "Trash Post"
- [ ] Update "Edit a Post"
- [ ] Update "Insert a contact form"
- [ ] Update "Insert a payment button"
- [ ] Update "Revert a post to draft"
- [ ] Update "Public Pages"
- [ ] Update "Private Pages"
- [ ] Update "Password Protected Pages"
- [ ] Update "Insert a payment button into a page"

cc @gwwar

**Comment by alisterscott** [comment]
Post editor in Gutenberg has been marked as a canary: #1623

**Comment by bsessions85** [comment]
Payment button block doesn't seem to be out there at this point

**Comment by alisterscott** [comment]
> Payment button block doesn't seem to be out there at this point

I think Payment Button and Contact Forms are still WIP with Jetpack

**Comment by alisterscott** [comment]
Private post mostly done in #1651

**Comment by alisterscott** [comment]
See https://github.com/Automattic/wp-e2e-tests/pull/1656 which converts most of the posts specs to run against Gutenberg in Calypso

**Comment by alisterscott** [comment]
@JavonDavis / @Stojdza I've listed three more items for Gutenberg in Calypso including the Simple Payments and Contact forms - feel free to work on any of these

**Comment by simison** [comment]
Noting that although Tiled gallery and Related posts block might be visible in Calypso-Gutenberg (behind the feature flag), they're not considered quite ready yet.

These should be "ready" (via p1HpG7-5V3-p2):

- Publicize
- Simple payments
- Forms 
- Maps
- Markdown

...although Maps is missing from Calypso currently. "Ready" in quotes because while they're ready in Jetpack, there are probably some rough corners in Calypso for us to fix in the next couple weeks since we've been mostly testing with wp-admin.

**Comment by JavonDavis** [comment]
> Add Contact Form Block test  

Investigating an issue I'm having with the Contact Form Block -> https://github.com/Automattic/wp-e2e-tests/issues/1640#issuecomment-441075452

**Comment by JavonDavis** [comment]
> > Add Contact Form Block test
> 
> Investigating an issue I'm having with the Contact Form Block -> [#1640 (comment)](https://github.com/Automattic/wp-e2e-tests/issues/1640#issuecomment-441075452)

This is still blocked and not ready on wpcom yet

**Comment by alisterscott** [comment]
> Even just having a test that ensures these blocks are not fatalling Calypso would be ace, though.

Good idea @simison - we've already added some tests for these blocks as they are

**Comment by simison** [comment]
>I've started a "related posts" test but it doesn't appear on the front end when the block is published - I will investigate why

That's expected — anything outside the "ready" list, namely related posts and tiled galleries are showing up in Calypso Gutenberg currently but cannot be considered ready at all.
I'll try to make sure you get heads up when these are ready for testing.
